finesse/__init__.py

```diff
@@ -1,17 +1,17 @@
 """The main module for the FINESSE program."""
 __version__ = "0.1.0"
 
-import sys
 
-from PySide6.QtWidgets import QApplication
+def run() -> None:
```

Comment:
I'm totally fine with this, but why did you moved all these imports within the function?

Reply:
There's more info in the commit message: 7b6ba3402587eb546437a05b7e36d926475fea72

The problem was that when I tried running the example code with `python -m finesse.hardware.tc4820` it was failing to import these modules. Putting the imports in `run()` fixed it, but I fear my Python-fu isn't good enough to understand what the root of the problem was :smile:

Reply:
Odd. Well, if it works... it's ok!